test/e2e/lib/helpers/broker_event_transformation_test_with_source.go

```diff
@@ -194,13 +194,13 @@ func BrokerEventTransformationTestWithPubSubSourceHelper(client *lib.Client, aut
 	time.Sleep(5 * time.Second)
 
 	// Create the PubSub source.
-	lib.MakePubSubOrDie(client,
-		lib.BrokerGVK,
-		psName,
-		brokerName,
-		topicName,
-		authConfig.PubsubServiceAccount,
-	)
+	lib.MakePubSubOrDie(client, lib.PubSubConfig{
+		SinkGVK:              lib.BrokerGVK,
+		PubSubName:           psName,
+		SinkName:             brokerName,
+		TopicName:            topicName,
+		SourceServiceAccount: authConfig.SourceServiceAccount,
```

Comment:
same everywhere, I'd rather use ServiceAccountName instead of SourceServiceAccount